draft-ietf-taps-interface.md

```diff
@@ -1045,26 +1045,53 @@ Type:
 : Enumeration
 
 Default:
-: Disabled
+: Disabled for connections created through initiate and rendezvous, Passive for listeners
 
-This property specifies whether an application wants to take advantage of
+This property specifies whether and how applications want to take advantage of
 transferring data across multiple paths between the same end hosts. Using
-multiple paths allows connections to migrate between interfaces as
-availability and performance properties change. Possible values are:
+multiple paths allows connections to migrate between interfaces or aggregate bandwidth
+as availability and performance properties change. Possible values are:
 
 Disabled:
-: The connection will not attempt using multiple paths once established
+: The connection will not allow using multiple paths once established
+
+Passive:
+: The connection will not actively try using multiple paths once established, but allow the peer endpoint to initiate using multiple paths and migrate existing connections to other paths.
 
 Handover:
-: The connection should attempt to migrate between different paths upon interface availability changes
+: The connection should only attempt to migrate between different paths when the original path is lost or becomes unreliable.
+The actual thresholds to declare a path unreliable are implementation specific.
```

Comment:
In #437, @tfpauly writes, as a conclusion from the interim discussion: "Exposing alternate addresses to the peer and actively migrating is likely a separate property that we should specify."  I interpret this as meaning that we don't want "Handover" here. If we do want to keep it, I agree with @gorryfair who, in #437, suggests to call this "Failover" instead.

Reply:
No, these are still two separate things:
 - Exposure of Alternative Addresses is defined as a separate property below
 - You don't need to expose an alternative addresses prior initiating handover.
 - Servers will most likely never advertise alternative addresses but still want to support the client in migrating.

Maybe make this more explicit that the Multipath property really defines path management behaviour while the exposure property defines advertisement.

Reply:
I saw the Exposure of Alternative Addresses below, and I agree that this is (or can be) separate from "actively migrating". My understanding of the sentence was that both of these things should be a separate property (or are somehow combined into a single separate property).

Reply:
@tfpauly Do you have a preference?

Reply:
It makes sense for a client to pick active, as ist allows to initiate additional subflows and, thus, implicitly expose some addresses, but not expose other addresses.
I consider exposing addresses without actively establishing sub-flows rather as a corner-case for things like GPT over SCPT or to bootstrap connection migration through NAT.
Note that I use MPTCP terminology here, but I think this is not limited to MPTCP.

Reply:
You can also be active and only establish connections if a new address is advertised by the peer.